.github/workflows/build-as-image.yml

```diff
@@ -0,0 +1,84 @@
+name: Build CoCo AS/RVPS Image
```

Comment:
The name here is different from `as-docker-build.yml`'s. What would this cause? idk do we need to make them the same?

Reply:
Hmm.. To my best knowledge, a title of the called workflow does not appear on an action section. I prefer to having a different title for each workflow. 😉